### PR TITLE
OSDOCS-9910:updates support limitations for InFW on AWS

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-9910
https://issues.redhat.com/browse/OSDOCS-9806

Link to docs preview:
https://75160--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ingress-node-firewall-operator.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

File: modules/nw-infw-operator-cr.adoc [hunk 1]
```diff
@@ -17,4 +17,6 @@ The Ingress Node Firewall Operator supports only stateless firewall rules.
 Network interface controllers (NICs) that do not support native XDP drivers will run at a lower performance.
 
 For {product-title} 4.14 or later, you must run Ingress Node Firewall Operator on {op-system-base} 9.0 or later.
+
+Ingress Node Firewall Operator is not supported on {aws-first} with the default OpenShift installation or on {product-rosa}. For more information on {product-rosa} support and ingress, see link:https://docs.openshift.com/rosa/networking/ingress-operator.html[Ingress Operator in Red Hat OpenShift Service on AWS].
```
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply modules/nw-infw-operator-cr.adoc:21]: I don't think putting the why in this section of the docs makes sense to me. I do however think that if we wanted to add a why we could create a bug and ship it with 4.16.0 and put the why in the release notes under bug fixes. 

@msherif1234 that work? All I need is a bug describing the issue and I'll take care of getting it in the RNs.

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Comment by mletalie** [comment]
Hello @JoeAldinger, just had one comment/suggestion, otherwise LGTM!

**Comment by JoeAldinger** [comment]
@mletalie hi, I can't see your suggestion.

**Comment by mletalie** [comment]
> @mletalie hi, I can't see your suggestion.
Sorry about that...was suggesting to show acronym as shown....
image

File: modules/nw-infw-operator-cr.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (msherif1234) [reply modules/nw-infw-operator-cr.adoc:21]: sound good to me Thanks!!

**Pull request merged**